flow/project/cli/project_test.go

```diff
@@ -311,3 +361,15 @@ func Test_ContractConflictComplex(t *testing.T) {
 	assert.False(t, notexists)
 
 }
+
+func Test_GetAliasesComplex(t *testing.T) {
+	p := generateAliasesProject()
+
+	aliases := p.GetAliases("emulator")
+	contracts := p.GetContractsByNetwork("emulator")
+
+	assert.Equal(t, 1, len(aliases))
+	assert.Equal(t, "ee82856bf20e2aa6", aliases["FungibleToken"])
+	assert.Equal(t, 1, len(contracts))
+	assert.Equal(t, "NonFungibleToken", contracts[0].Name)
```

Comment:
This is a small thing that I should have pointed out earlier, but all of the `assert` and `require` functions expect the argument order to be like this:

```
assert.Equal(t, EXPECTED, ACTUAL)
```

Although the order doesn't affect the correctness of the tests, it does change the failure messages (e.g. "expected 8 but got 12")

I'd recommend leaving this as-is for this PR and then changing all of the occurrences across all tests in a single separate PR.

Reply:
This is so weird. I initially had it reversed because it also makes sense that way but from the failure message, I realized I need to put it in current order. Not sure why I decided that. I will revert them no problem.

Reply:
I just tried to reverse it and I have this:

```
    account_test.go:43: 
                Error Trace:    account_test.go:43
                Error:          Not equal: 
                                expected: "f8d6e0586b0a20c7"
                                actual  : "f8d6e0586b0a20c1"
```

Test that intentionally fails:

```
assert.Equal(t, accounts.GetByName("test").Address.String(), "f8d6e0586b0a20c1")
```

Reply:
Fixed all this in aliases sources branch